pkg/reconciler/v1alpha1/revision/revision.go

```diff
@@ -455,48 +448,3 @@ func (c *Reconciler) updateStatus(desired *v1alpha1.Revision) (*v1alpha1.Revisio
 	existing.Status = desired.Status
 	return c.ServingClientSet.ServingV1alpha1().Revisions(desired.Namespace).UpdateStatus(existing)
 }
-
-// TODO(643) Change this logic in 0.5 to only drop the deprecated label
-//           Delete this logic in 0.6
```

Comment:
This is not what this change does.  cc @dprotaso

Reply:
The comments are wrong.

In 0.4 there is actually a functional dependency on the deprecated label. That's being dropped for 0.5 here: https://github.com/knative/serving/pull/3325

So the earliest time we could add code to remove the 'deprecated label' is 0.6 - then in 0.7 delete the removal logic.

@pivotal-joshua-rider and I were thinking it's not a real issue to have the deprecated label `/configurationMetadataGeneration` around given that it should be the same value as the non-deprecated label `/configurationGeneration`.

Then we don't need to make work for ourselves in 0.6 and 0.7.